Improving the Install page (#7443)
PostHog/posthog.com: contents/docs/integrate/_snippets/install-js-snippet.mdx

Reviewer: One thing is that this snippet gets reused across all products (such as https://posthog.com/docs/session-replay/installation), so it needs to be universal. I think it does a good enough job but just a reminder.

import Snippet from "../snippet.mdx"

> This is the simplest way to get PostHog up and running, and only takes a few minutes.

Reviewer 2: ```suggestion
This is the simplest way to get PostHog up and running. It only takes a few minutes.
```

#### Add to your website and product

> Copy the snippet below and replace `<ph_project_api_key>` and `<ph_instance_address>` with your project's values, then add it within the `<head>` tags of your product - ideally just before the closing `</head>` tag - on pages you want to track. 

Reviewer: ```suggestion
Copy the snippet below and replace `<ph_project_api_key>` and `<ph_instance_address>` with your project's values, then add it within the `<head>` tags at the base of your product - ideally just before the closing `</head>` tag. This ensures PostHog loads only any page users visit.
```
Most apps only have one <head> tag and PostHog does not need to be repeatedly added.

> Copy the snippet below and replace `<ph_project_api_key>` and `<ph_instance_address>` with your project's values, then add it within the `<head>` tags of your product - ideally just before the closing `</head>` tag - on pages you want to track. 

Author: Hm. Not sure I can see how we can accomodate that though. Probably best to leave it?

> You can find the snippet pre-filled with this data in [your project settings](https://app.posthog.com/project/settings).

Reviewer: ```suggestion
You can find the snippet pre-filled with this data in [your project settings](https://us.posthog.com/settings/project#snippet).
```

Author: Is US the right domain? I'm not sure if that will redirect to EU for relevant users

Reviewer: The US domain redirects 😄

<Snippet />

> Once the snippet has been added PostHog will automatically track `$pageview` and [other events](/docs/data/autocapture). You can then enable other products, such as session replays, within [your project settings](https://app.posthog.com/project/settings). 

Reviewer: ```suggestion
Once the snippet has been added, PostHog automatically captures `$pageview` and [other events](/docs/data/autocapture) like button clicks. You can then enable other products, such as session replays, within [your project settings](https://us.posthog.com/settings/project-replay). 
```

> Once the snippet is added, PostHog automatically captures `$pageview` and [other events](/docs/data/autocapture) like button clicks. You can then enable other products, such as session replays, within [your project settings](https://us.posthog.com/settings/project-replay). 

Author: ```suggestion
Once the snippet is added, PostHog automatically captures `$pageview` and [other events](/docs/data/autocapture) like button clicks. You can then enable other products, such as session replays, within [your project settings](https://us.posthog.com/settings). 
```

Actually, think we want this to be general as it isn't limited to just replay
